backtrace-library/src/main/java/backtraceio/library/BacktraceDatabase.java

```diff
@@ -114,6 +117,7 @@ public BacktraceDatabase(Context context, BacktraceDatabaseSettings databaseSett
         this.backtraceDatabaseFileContext = new BacktraceDatabaseFileContext(this.getDatabasePath(),
                 this.databaseSettings.getMaxDatabaseSize(), this.databaseSettings
                 .getMaxRecordCount());
+        this.backtraceBreadcrumbs = new BacktraceBreadcrumbs(getDatabasePath());
```

Comment:
@konraddysput see the new way of constructing Breadcrumbs which gets the database path.

Reply:
super coo, so instance on the creation time is disabled by default?

Reply:
Indeed. You must explicitly enable.